### ISSUE TITLE
Problem with building old source from the-new-hotness

### ISSUE THREAD

**Issue by phracek** [phracek/rebase-helper#182]
It seems, rebase-helper has a problem with building old sources
https://bugzilla.redhat.com/show_bug.cgi?id=1070102#c37
https://bugzilla.redhat.com/attachment.cgi?id=1157219
Can you please look on it somebody?

**Comment by phracek** [comment]
rebase-helper has to attach a failed build logs into bz, though.

**Comment by nforro** [comment]
Right, without the log from rpmbuild we can only guess where the problem is.
I suspect in some cases it could be caused by missing build dependencies, maybe we should run rpmbuild with *--nodeps* flag. I don't think that's the case with *ckeditor* though.
